__tests__/CollapsedMetrics.test.js

```diff
@@ -0,0 +1,611 @@
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 37, 1.833368, 200]
+                }, 
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 38, 1.821198, 250]
+                }, 
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 39, 1.08, 300]
+                },
+            ]
+        }
+            
+        let result = sut.getFirstRepROM(set);
+    
+        expect(result).toBe(200);
+    });
+});
+
+describe('getLastRepROM', () => {
+    
+    test('return null when avgVs empty', () => {
+        let set = {
+            reps: [],
+        };
+            
+        let result = sut.getLastRepROM(set);
+            
+        expect(result).toBe(null);        
+    });
+    
+    test('returns last rep avg velocity', () => {
+        let set = {
+            reps: [
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 37, 1.833368, 200]
+                }, 
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 38, 1.821198, 250]
+                }, 
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 39, 1.08, 300]
+                },
+            ]
+        };
+            
+        let result = sut.getLastRepROM(set);
+    
+        expect(result).toBe(300);
+    });
+});
+
+describe('getMinROM', () => {
+    
+    test('return null when roms empty', () => {
+        let set = {
+            reps: [],
+        };
+            
+        let result = sut.getMinROM(set);
+            
+        expect(result).toBe(null);        
+    });
+    
+    test('returns min rom', () => {
+        let set = {
+            reps: [
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 37, 1.833368, 200]
+                }, 
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 38, 1.821198, 250]
+                }, 
+                {
+                    isValid: true,
+                    removed: false,
+                    data: [-3456, 39, 1.08, 300]
+                },
+            ]
+        };
+            
+        let result = sut.getMinROM(set);
+    
+        expect(result).toBe(200);
+    });
+});
+
+describe('getMaxROM', () => {
+    
+    test('return null when avgVs empty', () => {
+        let set = {
+            reps: [],
+        };
+            
+        let result = sut.getMaxROM(set);
+            
+        expect(result).toBe(null);        
+    });
+    
+    test('returns max avg velocity', () => {
+        let set = {
+            reps: [
```

Comment:
- [ ]   modify test to have a invalid and removed values for "max" that are ignored